src/mdr/preprocess.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+from datetime import datetime
+from dateutil.parser import parse
+
+from src.models import MediaHouse, Status
+
+
+def preprocess_mdr_comment(raw: dict[str, Any]) -> dict[str, Any]:
+    """Preprocess comment.
+
+    :param raw: raw comment
+    """
+    return {
+        "id": raw["id"],
+        "status": Status.TO_BE_PROCESSED,
+        "body": raw["body"],
+        "asset_id": raw["asset_id"],
+        "asset_url": raw["asset"]["url"],
+        "author_id": raw["author_id"],
+        "username": raw["author"]["username"],
+        "created_at": parse(raw["created_at"]),
+        "last_updated_at": datetime.now(),
+        "mentions": None,
+        "media_house": MediaHouse.MDR,
+    }
```

Comment:
would it make sense to import the class Comment and substitute the return for:
Comment(raw["id"],
        Status.TO_BE_PROCESSED,
        raw["body"],
        raw["asset_id"],
        raw["asset"]["url"],
        raw["author_id"],
        raw["author"]["username"],
        parse(raw["created_at"]),
        datetime.now(),
        None,
        MediaHouse.MDR,
)

Reply:
Let's discuss this in person later. But in short: I don't see why the preprocessing should know what the data will be used for.